konfera/models/ticket_type.py

```diff
@@ -16,14 +28,47 @@
 
 class TicketType(FromToModel):
     title = models.CharField(max_length=128)
-    description = models.TextField()
+    description = models.TextField(blank=True)
     price = models.DecimalField(decimal_places=2, max_digits=12)
     attendee_type = models.CharField(choices=TICKET_TYPE_CHOICES, max_length=255, default='attendee')
     event = models.ForeignKey('Event')
 
     def __str__(self):
         return self.title
 
+    def _get_current_status(self):
+        now = timezone.now()
+        status = ACTIVE
+
+        if not self.date_from or not self.date_to or now < self.date_from:
```

Comment:
Is this check `not self.date_from or not self.date_to` necessary? I think every TicketType will have `date_from` because either the user sets it or the `clean` does. The same applies for `date_to`.